docs/man/git-lfs.1.ronn

```diff
@@ -82,3 +82,27 @@ commands and low level ("plumbing") commands.
     Git pre-push hook implementation.
 * git-lfs-smudge(1):
     Git smudge filter that converts pointer in blobs to the actual content.
+
+## EXAMPLES
+
+To get started with LFS, the following commands can be used.
+
+ 1. setup LFS in a git repository, this needs to be done only once:
```

Comment:
Maybe:
```
Setup Git LFS on your system. You only have to do this once:
```
Your version indicates that the user needs to do this for every repo.

Reply:
i thought you *did* have to run `git lfs install` in each repository. is that incorrect? how are hooks setup otherwise?

Reply:
That's correct, and I think we can extend @larsxschneider's idea to capture this, perhaps:

> Setup Git LFS on your system. You only have to do this once per repository per machine: